spec/lib/sentry/processor/filter_request_body_spec.rb

```diff
@@ -3,47 +3,54 @@
 require 'rails_helper'
 
 RSpec.describe Sentry::Processor::FilterRequestBody do
-  let(:client) { double('client') }
-  let(:processor) { Sentry::Processor::FilterRequestBody.new(client) }
-  let(:result) { processor.process(sentry_data) }
+  context 'with PII in the [:request][:data] hash' do
+    before(:each) do
+      client = double('client')
+      @processor = Sentry::Processor::FilterRequestBody.new(client)
```

Comment:
I'm tempted to just inline `double('client')` here since it doesn't affect anything

Reply:
Only argument against I can think of is if someone didn't notice and duplicated that could be avoided by leaving it at top of block.